db-es-audit/src/find_granule_counts.py

```
@@ -0,0 +1,616 @@
+    Get the list of granule indexes in elastic search.
+
+    Args:
+        None
+    
+    Returns:
+        The list of granule indexes.
+
+    Exceptions:
+        None
+    """
+    es_indices_response = requests.get(f"{GRAN_ELASTIC_URL}/_cat/aliases?h=alias&format=json")
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)